source/Ocl/OclSerializer.cs

```diff
@@ -8,9 +8,12 @@ public interface IOclSerializer
     {
         string Serialize(OclDocument document);
         string Serialize(object obj);
-        OclDocument ToOclDocument(object? obj);
+
+        OclDocument Deserialize(string str);
```

Comment:
Can I not just call `Deserialize<OclDocument>(`?

Reply:
Turns out you can... 🤦